app/components/primer/beta/auto_complete/item.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
-
+# TODO: use generic ActionList item for Autocomplete
 module Primer
   module Beta
     class AutoComplete
       # Use `AutoCompleteItem` to list results of an auto-completed search.
       class Item < Primer::Component
+        include ViewComponent::PolymorphicSlots
```

Comment:
Does this component use polymorphic slots?

Reply:
I have no idea 🙃 😬

Reply:
It doesn't look like it does, so we can probably remove this `include`.